manifests/pod.yaml

```diff
@@ -28,6 +28,8 @@ spec:
          value: /.gcp-image-upload-config/config
        - name: GCP_KOLA_TESTS_CONFIG
          value: /.gcp-kola-tests-config/config
+       - name: OPENSTACK_KOLA_TESTS_CONFIG
+         value: /.openstack-kola-tests-config/config
```

Comment:
Hmm, we shouldn't need to touch `pod.yaml` if it's just the `kola-openstack` job that uses it (and actually, we should remove the GCP and AWS test secrets as well from there).

Reply:
we ended up needing to because the main Jenkinsfile cues running the tests off of them 

https://github.com/coreos/fedora-coreos-pipeline/blob/390a7f45d3cec9dc70f8911e13424adbbbdfe9e6/Jenkinsfile#L499